worker/fbs/rtpParameters.fbs

```diff
@@ -9,11 +9,11 @@ table Boolean {
   value:uint8;
 }
 
-table Integer {
+table Integer32 {
```

Comment:
even `Int32` could make it.

Reply:
No, please. If so I should also rename `Integer32Array` to `Int32Array`. However `Int32Array` is a JS binary array class so conflicts.